addons/addon-base-raas/packages/base-raas-services/lib/costs/costs-service.js

```diff
@@ -137,9 +140,10 @@ class CostsService extends Service {
     const [aws] = await this.service(['aws']);
     const { accessKeyId, secretAccessKey, sessionToken } = await this.getCredentials(requestContext, indexId);
 
+    const region = this.settings.get(settingKeys.awsRegion);
     const costExplorer = new aws.sdk.CostExplorer({
       apiVersion: '2017-10-25',
-      region: 'us-east-1',
+      region,
```

Comment:
Only found ce endpoint for cn-northwest-1, is it workable for cn-north-1?

https://docs.amazonaws.cn/en_us/aws/latest/userguide/endpoints-Beijing.html

Reply:
yes, check and verify it. cn-north-1 and cn-northwest-1 should have the same endpoint: ce.cn-northwest-1.amazonaws.com.cn